network/network_test.go

```diff
@@ -971,3 +1055,797 @@ func TestTrackConnectedRace(t *testing.T) {
 	err = net1.Close()
 	assert.NoError(t, err)
 }
+
+func assertEqualPeers(t *testing.T, expected map[string]ids.ShortID, actual []PeerID) {
+	assert.Len(t, actual, len(expected))
+	for _, p := range actual {
+		match, ok := expected[p.IP]
+		assert.True(t, ok, "peer with IP %s missing", p.IP)
+		assert.Equal(t, match.PrefixedString(constants.NodeIDPrefix), p.ID)
+	}
+}
+
+func TestPeerAliases_Ticker(t *testing.T) {
```

Comment:
nit: remove the underscore in the function name to follow the syntax of the rest of the tests